src/ebonite/client/base.py

```diff
@@ -59,6 +60,25 @@ def push_model(self, model: 'core.Model', task: 'core.Task' = None) -> 'core.Mod
         model = self.meta_repo.save_model(model)
         return model
 
+    def delete_model(self, model: 'core.Model', ignore_artifact_failure=False):
+        """
+        Deletes :py:class:`~ebonite.core.objects.Model` instance from metadata and artifact repositories
+
+        :param model: model instance to delete
+        :param ignore_artifact_failure: whether model artifacts' deletion errors should be ignored, default is false
+        """
+        if model.artifact is not None:
+            try:
+                self.artifact_repo.delete_artifact(model)
+            except Exception as e:  # noqa
+                if ignore_artifact_failure:
+                    logger.warn(f"Unable to delete artifacts associated with model: '{model}'", e)
```

Comment:
I use "%s", xxx  for loggers instead of f"{xxx}" for laziness

Reply:
agree, fixed and simplified exception thing